prowler/providers/aws/services/accessanalyzer/accessanalyzer_service.py

```diff
@@ -43,8 +43,8 @@ def _list_analyzers(self, regional_client):
             if analyzer_count == 0:
                 self.analyzers.append(
                     Analyzer(
-                        arn=self.audited_account_arn,
-                        name=self.audited_account,
+                        arn=self.get_unknown_arn(regional_client.region),
```

Comment:
What if we add here an ARN with the following format:
`arn:${Partition}:access-analyzer:${Region}:${Account}:analyzer/unknown` 

so you can call it with `get_unknown_arn(region=region, resource_id="analyzer/unknown")`

Reply:
Done